### DISCUSSION TITLE
Hello and Welcome everyone

### DISCUSSION THREAD

**Discussion by vinodsharma10x** [careerJumpstart/TestimonialFree#2] [General]
Let's connect to start talking about this project.

Following is a list of people interested in working on this project. 

Please feel free to add your name to this list.

1. https://twitter.com/vishrut19
2. https://twitter.com/geekazeez
3. https://twitter.com/roramigator
4. https://twitter.com/tkbharadwaj
5. https://twitter.com/saimurugeshwari
6. https://twitter.com/Aaushhhhhhh

Following are a few mentors I will reach out to help 
1. https://twitter.com/ashokpatidar
2. https://twitter.com/toby_solutions
3. https://twitter.com/prashanttgs

**Comment by roberto-morado** [comment]
I will be more than glad; what are the next steps?

**Reply by vinodsharma10x** [reply]
I created a list of the next step under the project. 

https://github.com/orgs/careerJumpstart/projects/1

My goal is by this weekend, 

1/https://github.com/orgs/careerJumpstart/projects/1?pane=issue&itemId=18165386
2/Create high level technical design - database structure

**Reply by roberto-morado** [reply]
It seems to be an problem. I'm getting 404 errors on both links

**Reply by vinodsharma10x** [reply]
Sorry about that, i updated the project setting and now it should be visible.

**Reply by roberto-morado** [reply]
Looks awesome!

**Comment by ClaraLopezUribe** [comment]
Hello Vinod. I would love to contribute to this project in any way I can. Thank you for creating this great opportunity to learn as a community! 

How can I add my name to the list above?
